bionic/dagviz.py

```diff
@@ -4,16 +4,53 @@
 is also required.
 """
 
+import warnings
+from pathlib import Path
 from collections import defaultdict
-from io import BytesIO
+from io import BytesIO, IOBase
 
-from .optdep import import_optional_dependency
+from .optdep import import_optional_dependency, oneline
```

Comment:
super nit: let's import `oneline` from `util` since that's where it's "supposed" to be.